start.go

```diff
@@ -17,25 +17,19 @@ func startCmd() cli.Command {
 		Usage:  "start a functions server",
 		Action: start,
 		Flags: []cli.Flag{
-			cli.StringFlag{
```

Comment:
Why did you remove this?

Reply:
because it's not necessary anymore, you can put one to env file or just set FN_LOG_LEVEL to DEBUG and you'll be fine.

Reply:
How would FN_LOG_LEVEL make it into the container?

Reply:
through env file, that's the purpose of this whole patch - let user pass fn server opts to the container.

Reply:
Ok, well please don't make .env file required, you're making a lot of assumptions here.

Reply:
it's not required by default

Reply:
I'm saying, please keep log-level there. I don't want to have to make an env file to get debugging output.